airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1237,6 +1237,31 @@ components:
           type: boolean
         securityUpdates:
           type: boolean
+        notifications:
+          type: array
+          items:
+            $ref: "#/components/schemas/Notification"
+    Notification:
+      type: object
+      required:
+        - notificationType
+      properties:
+        # Instead of this type field, we would prefer a json schema "oneOf" but unfortunately,
```

Comment:
If that is the case, we should just have one field for each type and the overarching type on top:
```
Notification:
  type: object
  required:
    - notificationType
  properties:
    notificationType:
      $ref: "#/components/schemas/NotificationType"
    slackConfig:
      $ref: "#/components/schemas/SlackConfig"
    somethingElseConfig:
      $ref: "#/components/schemas/SomethingElseConfig"
```